chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -206,6 +311,16 @@ class CurrentUser(
             callback: (Result<List<Message>, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.fetchMessages(roomId, initialId, direction, limit) }, callback)
 
+    /**
+     * Fetches multipart messages for a [roomId] from an optional [initialId]. You can specify the direction
+     * of messages you want to receive, by default this is [Direction.OLDER_FIRST] which means you get the older messages
+     * through first. However you can alternatively request the messages in [Direction.NEWER_FIRST] which means
+     * you'd get the newer messages first. You can specify how many messages to receive back in the [limit],
+     * by default 10 messages will be returned. The [callback] will be called with a [Result] when the operation
+     * is complete. If the operation was successful you will receive an
+     * [List<com.pusher.chatkit.messages.multipart.Message>] which contains the message id,
+     * or an [Error] informing you of what went wrong.
```

Comment:
This paragraph is pretty long and could do with some additional structure.

It's not clear what "from an optional [initialId]" means.

It might be clearer to be less conversational here where there is a lot of structured information to convey.

Reply:
I've updated the format to be like the createRoom method documentation:

     * Fetches multipart messages for a [roomId] from an optional message [initialId].
     * - [roomId] - the room id you want to get messages for
     * - [initialId] - optional - the message id for the [0] message, if no initialId is provided the [0] message is the last message sent to the room
     * - [direction] - optional - the direction messages are ordered in the list, by default this is [Direction.OLDER_FIRST], alternatively you can request the messages in [Direction.NEWER_FIRST]
     * - [limit] - optional - the number of messages you want to receive, by default this will be 10 messages.
     * - [callback] - returns a [Result] with the [List<com.pusher.chatkit.messages.multipart.Message>] if successful, or an [Error] letting you know what went wrong
 
Should I explain more what `Direction.OLDER_FIRST` and `Direction.NEWER_FIRST` mean or do you think that's explained with the enum?